README_template.md

```diff
@@ -0,0 +1,40 @@
+# Project Name
+One paragraph description of the project goes here.
+
```

Comment:
Add some text to say details should be included of who the proejct was for/ what it was used for.  Who was the primary customer?